docs/rules/no-shadow.md

```diff
@@ -65,34 +65,32 @@ function foo() {
 }
 ```
 
-### `hoist`
+### hoist
 
-The option has three settings:
+The `hoist` option has three settings:
 
-* `all` - reports all shadowing before the outer variables/functions are defined.
 * `functions` (by default) - reports shadowing before the outer functions are defined.
+* `all` - reports all shadowing before the outer variables/functions are defined.
```

Comment:
Yes, so you can find the default value more easily and contrast it with the alternatives more clearly. Through the work on examples, we will learn about the patterns for describing options.